### PR TITLE
Think it is safe to depend on vSphere 5.0 minimum

### PR DESCRIPTION
**Pull request by agrare**
vSphere 5.0 was released in 2011, think we can drop code supporting 2.x
and 4.x

### CODE REVIEW

File: lib/VMwareWebService/VimService.rb [hunk 1]
```diff
@@ -26,6 +26,8 @@ def initialize(ep)
     @v20             = @apiVersion =~ /2\.0\..*/
     @v2              = @apiVersion =~ /2\..*/
```
Comment (roliveri) [inline lib/VMwareWebService/VimService.rb]: Do we still need `@v20` and `@v2`?

Comment (agrare) [reply lib/VMwareWebService/VimService.rb]: I was going to delete them but didn't want to break the API until I made sure no one was using them

Comment (jerryk55) [reply lib/VMwareWebService/VimService.rb]: And 4 as well based on your original comment, I assume?

Comment (roliveri) [reply lib/VMwareWebService/VimService.rb]: Right, it's probably better to keep them for now.

**Pull request merged**